vowpalwabbit/core/src/parse_args.cc

```diff
@@ -408,6 +408,48 @@ input_options parse_source(VW::workspace& all, options_i& options)
                      "hashed as A^B^C."))
       .add(make_option("flatbuffer", parsed_options.flatbuffer)
                .help("Data file will be interpreted as a flatbuffer file")
+               .experimental())
+      .add(make_option("csv", parsed_options.csv).help("Data file will be interpreted as a csv file").experimental())
+      .add(make_option("csv_separator", all.csv_separator)
+               .help("CSV Parser: Explicitly specify field separator in one character "
+                     "Default: ',' (comma)")
+               .experimental())
+      .add(make_option("csv_ns_separator", all.csv_ns_separator)
+               .help("CSV Parser: Explicitly specify separator for namespace and feature name of the header in "
+                     "one character. We will only separate from the first ns_separator, if no separator exists, "
+                     "then the namespace would be empty. "
+                     "Default: '.'")
```

Comment:
Default is a |, right? Also, can't we specify the default value in the `add()` call here instead of having the extra logic in `handling_separator()`?

Reply:
> Default is a |, right?

Oh, I forgot to update that in the help message.

> Also, can't we specify the default value in the add() call here instead of having the extra logic in handling_separator()

Yeah, we can do that. I didn't notice that those options can't be empty, so will remove the string empty branch  in handling_separator() and add the default_value() in the add() call.